.circleci/config.yml

```diff
@@ -28,7 +28,7 @@ jobs:
             fi
       - run:
           name:  "Test: Execute Python Linting Test"
-          command: "cd tdrs-backend; docker-compose run --rm web bash -c \"flake8 .\""
+          command: "cd tdrs-backend; docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm web bash -c \"flake8 .\""
       - run:
           name:  "Test: Execute Python OWASP ZAP Vulnerability Scan"
           command: "cd tdrs-backend; ./zap-scanner.sh"
```

Comment:
@alexsoble @hursey013 Changes to this file were necessary to accomplish the goal of this task. Previously we were using a `.env` file to load the variables that CircleCI was using. Since one of the objectives of this PR was to add `.env` files to `.gitignore` the way we deliver variables to CircleCI had to change.

Reply:
Note: see latest changes made in tdrs-backend/tdpservice/settings/common.py

With those calls to `os.environ.setdefault` we are able to run this within CI effectively just as it was before. The `JWT_KEY` and `JWT_CERT_TEST` values will still get passed in from CI context via the base docker-compose but defaults are defined for all non-sensitive variables in the settings file now so we no longer need the additional `docker-compose.ci.yml` file nor do we need to define those variables in CircleCI project settings.

By updating this CI config to manually pass the compose file we are just ensuring that CircleCI doesn't attempt to pass an `env_file` in.